Library/Homebrew/formula.rb

```diff
@@ -23,6 +23,7 @@
 require "tab"
 require "mktemp"
 require "find"
+require "byebug"
```

Comment:
Sorry, still can't include it even here 😭. Would need to just be in various `dev-cmd` after `Homebrew.install_bundler_gems!`

Reply:
@MikeMcQuaid I've made some changes to `dev-cmd/tests` for now, and if the approach looks good, I'll go ahead and do this to other commands as well.